src/main/kotlin/ch/wisv/choice/document/service/DocumentServiceImpl.kt

```diff
@@ -51,7 +50,7 @@ class DocumentServiceImpl(val documentRepository: DocumentRepository,
     }
 
     override fun getDocumentBytesById(id: Long): ByteArray {
-        val document = documentRepository.findOne(id) ?: throw CHoiceException("Document with id $id does not exists!")
+        val document = documentRepository.findById(id).orElse(null) ?: throw CHoiceException("Document with id $id does not exists!")
```

Comment:
Aangezien je die andere wijziging gerevert hebt, hier ook?

Reply:
Deze wijziging is wel nodig. De jparepository heeft het interface van findOne verandert. `findById` doet nu findOne met nog een option eromheen dus vandaar die orElse.